fleetd/fleet.go

```diff
@@ -213,6 +222,33 @@ func listenForSignals(sigmap map[os.Signal]func()) {
 	}
 }
 
+func createPidFile(pidfile string) error {
+	if pidstring, err := ioutil.ReadFile(pidfile); err == nil {
+		pid, err := strconv.Atoi(string(pidstring))
+		if err == nil {
+			if _, err := os.Stat(fmt.Sprintf("/proc/%d/", pid)); err == nil {
```

Comment:
@scole-scea , Thanks for your suggestion. 
Have modify to check "/proc/%d/comm" whether it's "fleetd" also. 
Actually, if one process exits, the pid is reused again, need wait a long time. And the process name check can void the other process to use the same pid. 
Please refer to my another PR #1229